lib/vrf.c

```diff
@@ -944,8 +944,9 @@ static int lib_vrf_create(struct nb_cb_create_args *args)
 		return NB_OK;
 
 	vrfp = vrf_get(VRF_UNKNOWN, vrfname);
-
-	SET_FLAG(vrfp->status, VRF_CONFIGURED);
+	
+	if (vrfp)
```

Comment:
We should return a warning/error instead then? Also, check `vrf_handler_create()`.